prov/verbs/configure.m4

```diff
@@ -11,6 +11,7 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 	# Determine if we can support the verbs provider
 	verbs_ibverbs_happy=0
 	verbs_rdmacm_happy=0
+	verbs_raifamily_happy=0
```

Comment:
You should only need a single rdmacm_happy check that looks for the correct version.